### ISSUE TITLE
Expose Non-Connectable Advertising

### ISSUE THREAD

**Issue by miwright2** [JuulLabs/kable#210]
Similar to Issue #65.  Is it possible to expose the connectable state of an advertising packet?  For example:

```
public actual class Advertisement(
    private val scanResult: ScanResult,
) {

    internal val bluetoothDevice: BluetoothDevice
        get() = scanResult.device

    public actual val isConnectable: Boolean
        get() = scanResult.isConnectable
...
}
```

[Similar information on iOS.](https://developer.apple.com/documentation/corebluetooth/cbadvertisementdataisconnectable)

**Comment by twyatt** [comment]
Seems easy enough. Thanks for the link to iOS docs. I'll have to check if Web Bluetooth also offers this boolean, to allow for a common API across all the platforms.

**Comment by twyatt** [comment]
Available in [0.21.0](https://github.com/JuulLabs/kable/releases/tag/0.21.0).
